python-main.js

```diff
@@ -154,6 +157,12 @@ function web_editor(config) {
     // Set the font size of the text currently displayed in the editor.
     function setFontSize(size) {
         EDITOR.ACE.setFontSize(size);
+        $("#request-repl")[0].style.fontSize = "" + size + "px";
+
+        // Only update font size if REPL is open
+        if ($("#repl").css('display') != 'none') {
```

Comment:
I haven't tested this, but if the font size is only changed when the REPL is opened this will:
- No change the font size of the REPL when viewing the code
- Change the code font size while viewing the REPL

Which is inconsistent.

Reply:
They both maintain the same font size, it is just when you switch from one to the other it will show at its previous font size for >1 second before updating to the current font size. It appears the code editor already had this behavior from when you exit the REPL anyway so it is more consistent now.

Reply:
Okay, sounds good then, but just to double check if I were to:

- Open the editor
- Increase font size a couple of times
- Open the REPL

Then the hterm font will be a couple of steps larger than the default?

And if in the same session I:

- Increase font size a couple of times
- Close the REPL

Then the text in the code editor would be a 4 steps larger than the default?

Reply:
Yes, that is correct!

Reply:
Great, thanks!